components/InputStates.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * InputStates.js
+ * 
+ * Manages the input states for a component's ports
+ */
+
+var _ = require('underscore');
+
+var inputStates = [];
```

Comment:
No worries.  And I have addressed the issue of easily integrating it into the vni in a check in I will make shortly.